tests/unit_tests/blas/level1/asum_usm.cpp

```diff
@@ -86,7 +86,9 @@ int test(const device& dev, int64_t N, int64_t incx) {
 #else
         TEST_RUN_CT(main_queue, onemkl::blas::asum,
                     (main_queue, N, x.data(), incx, result_p, dependencies));
+    #ifndef ENABLE_CUBLAS_BACKEND
         main_queue.wait();
```

Comment:
Why do we wait on the queue for the USM tests and not on the returned event?

Reply:
This is due to this issue (intel/llvm#1569).
However, I just found out that, the issue is fixed in recent builds, so I will test and update the code. (remove #ifndef ENABLE_CUBLAS_BACKEND).

Reply:
Line removed.